src/compaction_filter.rs

```diff
@@ -282,7 +298,7 @@ mod tests {
     use super::{
         CompactionFilter, CompactionFilterContext, CompactionFilterFactory, DBCompactionFilter,
     };
-    use crate::{ColumnFamilyOptions, DBOptions, DB};
+    use crate::{new_compaction_filter_raw, ColumnFamilyOptions, DBOptions, Writable, DB};
```

Comment:
Not formatted it seems?

Reply:
Oh I missed it. And I just found I can remove `KeyRangeFilter` in the test.

Reply:
I reviewed the code, seems everything imported here is used.

Reply:
I mean they are not sorted